src/pages/NotFound.js

```diff
@@ -1,12 +1,12 @@
 import React, { Component } from "react";
 
+import ErrorComponent from "../components/Errors/ErrorComponent";
+
 class NotFound extends Component {
 
     render() {
         return (
-            <div>
-                Oh no! Someone took this page away!
-            </div>
+            <ErrorComponent Message={"Sorry, no page could be found at this address (404)"}/>
```

Comment:
If you add a title prop to the error component, you can keep it generic and pass a different title and message for 404 and unexpected errors